### DISCUSSION TITLE
Any plans to implement adding shapes (formatted textboxes)?

### DISCUSSION THREAD

**Discussion by Knexxxx** [dolanmiu/docx#1573] [Ideas / Requests]
Docx has really a 99% coverage of the features. really great. What I would love to see is the possibility to create shapes like a circle or a square and add text to it. I "only" found adding textboxes so far. Do you have plans to implement that feature or will that be almost impossible? 
It would be great to write "pill-shaped-text" using docx...

**Comment by dolanmiu** [comment]
Potentially possible yes, been reading about how it is done, it seems it is just a textbox with a graphicData element:

http://officeopenxml.com/drwOverview.php

The complexity lies in the graphicData part I think
